class.eventcalendar.plugin.php

```diff
@@ -399,11 +414,12 @@ public function vanillaController_eventCalendar_create($sender, $args = []) {
         $sender->setData('PreviousMonth', date('Y', $monthFirst - 1).'/'.date('m', $monthFirst - 1));
         $sender->setData('NextMonth', date('Y', $monthLast + 86400).'/'.date('m', $monthLast + 86400));
         $sender->setData('DaysInMonth', $daysInMonth);
-
-        $sender->setData('Events', $eventCalendarModel->get("{$year}-{$month}-01", "{$year}-{$month}-{$daysInMonth}"));
+		
+		$plusUn = $month+1;
```

Comment:
Mixing language in source code is a bad idea. Furthermore there should be a blank before and after the plus sign. Vanilla normally uses speaking variables. Looking at a variable called "plus one" gives no hint about its meaning

Reply:
yeah i don't understand the syntax you're using just the line after